### PR TITLE
Internationalization!

### PR DESCRIPTION
**Pull request by hemisemidemipresent**
after like more than 3 years, #13 has been implemented

### CODE REVIEW

**Comment by hemisemidemipresent** [comment]
turning off waveform doesnt work unfortunately

**Comment by Jasonnor** [comment]
Great and inspiring feature! I will review ASAP 😆

File: scripts/i18n.js [hunk 1]
```diff
@@ -0,0 +1,58 @@
+let cached = {};
+let langSelect = document.getElementById('langSelect');
+function getLangfile(lang) {
+    return new Promise(function (resolve, reject) {
+        if (cached[lang]) resolve(cached[lang]);
+        axios
+            .get(
+                'https://www.thpatch.net/w/index.php?title=Special:Translate&taction=export&group=themedb&language=' +
```
Comment (Jasonnor) [inline scripts/i18n.js:8]: Request will been blocked by CORS. Maybe we can only maintain those json files in local files or Firebase?

image

**Review by Jasonnor — CHANGES_REQUESTED** [review]
Did you encounter CORS issues when testing locally? If not, maybe we can discuss how to avoid it. 😃

File: scripts/i18n.js (hunk 1 above)
Comment (hemisemidemipresent) [reply scripts/i18n.js:8]: yeah if it could be done through firebase that would be ideal

Comment (hemisemidemipresent) [reply scripts/i18n.js:8]: I didnt get any CORS issue when running with live server locally though

Comment (Jasonnor) [reply scripts/i18n.js:8]: Got it! I will update Firebase & those code to get json files.

Comment (hemisemidemipresent) [reply scripts/i18n.js:8]: actually the code in production appears to work fine

Comment (Jasonnor) [reply scripts/i18n.js:8]: Wow, but I tried to deploy to GitHub Pages earlier with this branch and still got the same error, maybe a browser limitation? 

I tested in Chrome 97.0.4692.99. Maybe I should deploy again to verify it!

Comment (Jasonnor) [reply scripts/i18n.js:8]: Building: https://github.com/Jasonnor/th-music-video-generator/actions/runs/1746537176

Comment (hemisemidemipresent) [reply scripts/i18n.js:8]: ah if its the browser specific then it will be better to take it from firebase

Comment (hemisemidemipresent) [reply scripts/i18n.js:8]: how is it going on the firebase end?

Comment (Jasonnor) [reply scripts/i18n.js:8]: I might be working on it on weekends, it's been a bit busy lately LOL 😄

**Review by Jasonnor — APPROVED** [review]
LGTM

**Pull request merged**